docs/enriching-your-data/managing-enrichments/ui/index.md

```diff
@@ -1,7 +1,11 @@
 ---
 title: "Managing enrichments in BDP Console"
 date: "2020-02-15"
+sidebar_label: "Using the UI"
 sidebar_position: 10
+sidebar_custom_props:
+  offerings:
+    - enterprise
```

Comment:
How would you feel about updating the gifs on this page, they are very out of date?